shell/platform/fuchsia/flutter/component.cc

```diff
@@ -497,6 +497,11 @@ Application::Application(
   settings_.task_observer_remove = std::bind(
       &CurrentMessageLoopRemoveAfterTaskObserver, std::placeholders::_1);
 
+  settings_.log_message_callback = [](const std::string& tag,
+                                      const std::string& message) {
+    std::cout << tag << ": " << message << std::endl;
```

Comment:
@arbreng @freiling this matches the current behaviour, so keeping as-is in this patch to avoid conflating functional changes with the refactoring. If there's a different way we should be doing this on Fuchsia, let me  know. One key place this is used is for logging the observatory URL the tool uses for hot-reload -- I'm not familiar with Fuchsia's tooling around that today though.